frontend/scripts/react-components/shared/profile-selector/profile-selector.actions.js

```diff
@@ -109,10 +74,61 @@ export const setProfilesLoadingItems = loadingItems => ({
     loadingItems
   }
 });
+export const goToProfile = () => (dispatch, getState) => {
+  const { profileSelector, app } = getState();
+  const { contexts } = app;
+  const hasCompanies = !isEmpty(profileSelector.panels.companies.activeItems);
+  const profileType = hasCompanies ? 'actor' : 'place';
+  const profileSelection = hasCompanies
+    ? Object.values(profileSelector.panels.companies.activeItems)[0]
```

Comment:
why not use a activeItem instead of an array

Reply:
I tried but we are reusing the panels from dashboards and it would be even more confusing there

Reply:
I see, you're right. But then the CompaniesPanel prop should be `activeCompanies` instead of `activeCompany`